modules/javafx.base/src/main/java/javafx/beans/binding/Bindings.java

```diff
@@ -161,11 +161,17 @@ protected boolean computeValue() {
                 }
             }
 
+            /**
+             * Calls {@link BooleanBinding#unbind(Observable...)}.
+             */
             @Override
             public void dispose() {
                 super.unbind(dependencies);
             }
 
+            /**
+             * Returns an immutable list of the dependencies of this binding.
+             */
```

Comment:
Does the missing `@return` tag matter here? It isn't public docs so might not matter. The same question applies to the other instances of this overridden method.

Reply:
I don't think it matters. The description is more important.